src/test/java/com/github/onsdigital/JWTKeyProviderImplTest.java

```diff
@@ -27,18 +27,20 @@
     Map<String, String> signingKeys = new HashMap<String, String>() {{
         put(PUBLIC_KEY_ID, PUBLIC_KEY);
     }};
+    private String identityAPIUrl = "http://localhost:8080";
 
     @Test
     void verify_ShouldThrowException_WhenIdentityApiReturnEmptySuccessResponse() throws Exception {
         HttpResponse mockResponse = Mockito.mock(HttpResponse.class);
         given(mockResponse.getStatusCode()).willReturn(200);
         Mockito.when(mockResponse.parseAs(Mockito.any())).thenReturn(new HashMap<String, String>());
         RequestBuilder mockedRequestBuilder = getMockedRequestBuilder(mockResponse);
-        JWTKeyProviderImpl jwtKeyProvider = new JWTKeyProviderImpl("url", 5, 5, 5, mockedRequestBuilder);
+        JWTKeyProviderImpl jwtKeyProvider = new JWTKeyProviderImpl(identityAPIUrl, 5, 5, 5, mockedRequestBuilder);
 
         Exception exception = assertThrows(Exception.class, () -> jwtKeyProvider.getJwtKeys());
         Assert.assertThat(exception.getMessage(), CoreMatchers.containsString("JWT keys not found in the response"));
         verify(mockResponse, times(1)).disconnect();
+        verify(mockedRequestBuilder).getRequest("http://localhost:8080/v1/jwt-keys", 5, 5, 5);
```

Comment:
Can we move `"http://localhost:8080/v1/jwt-keys"` to a `private static final String` and use that in calls to `getRequest`?